modules/ref_argocd-appset-create.adoc

```diff
@@ -0,0 +1,35 @@
+// Module is included in the following assemblies:
+//
+// * gitops_cli_argocd/argocd-gitops-cli-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="argocd-appset-create_{context}"]
+= argocd appset create
+
+Create `ApplicationSet` resources for your application.
+
+.Command syntax
+[source,terminal]
+----
+argocd appset create [flags]
+----
+
+.Example - Create a new filename or a URL for an `ApplicationSet` resource using the following command:
+
+[source,terminal]
+----
+Create ApplicationSets
```

Comment:
`Create ApplicationSets` should not be part of the command

Reply:
Fixed.